code/modules/clothing/glasses/glasses.dm

```diff
@@ -478,19 +478,21 @@
 		"Grey" = 'icons/mob/clothing/species/grey/eyes.dmi',
 		"Drask" = 'icons/mob/clothing/species/drask/eyes.dmi'
 		)
-
-/obj/item/clothing/glasses/godeye/attackby(obj/item/W as obj, mob/user as mob, params)
-	if(istype(W, src) && W != src && W.loc == user)
-		if(W.icon_state == "godeye")
-			W.icon_state = "doublegodeye"
-			W.item_state = "doublegodeye"
-			W.desc = "A pair of strange eyes, said to have been torn from an omniscient creature that used to roam the wastes. There's no real reason to have two, but that isn't stopping you."
+	///Have we combined this with another godeye?
+	var/double_eye = FALSE
+
+/obj/item/clothing/glasses/godeye/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/clothing/glasses/godeye) && W != src)
+		var/obj/item/clothing/glasses/godeye/second_eye = W
+		if(!second_eye.double_eye && !double_eye)
+			double_eye = TRUE
+			icon_state = "doublegodeye"
+			item_state = "doublegodeye"
+			desc = "A pair of strange eyes, said to have been torn from an omniscient creature that used to roam the wastes. There's no real reason to have two, but that isn't stopping you."
```

Comment:
Minor thing, but could you also change their name to `eyes of god`? Since it's a pair.